_translator-files/po/nb_NO/Client-Troubleshooting.po

```diff
@@ -1,31 +1,31 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Allan Nordhøy <epost@anotheragency.no>, 2022.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2022-10-23 13:06+0000\n"
+"Last-Translator: Allan Nordhøy <epost@anotheragency.no>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language: nb_NO\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.14.2-dev\n"
 
 #. type: YAML Front Matter: lang
 #: ../wiki/en/Client-Troubleshooting.md:1
-#, no-wrap
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```

Comment:
@comradekingu please use the full language code here (nb_NO).

Reply:
```suggestion
msgstr "nb_NO"
```

Reply:
nb_NO now in https://hosted.weblate.org/translate/jamulus/client-troubleshooting/nb_NO/?checksum=cc3a3b8aeba83df4